src/Microsoft.Identity.Web/TokenAcquisitionOptions.cs

```diff
@@ -46,6 +47,11 @@ public class TokenAcquisitionOptions
         /// </summary>
         public PoPAuthenticationConfiguration? PoPConfiguration { get; set; }
 
+        /// <summary>
+        /// todo.
+        /// </summary>
+        public CancellationToken CancellationToken { get; set; } = CancellationToken.None;
```

Comment:
As stated above, this is a non-standard way of exposing cancellation token. Best to put it as the last param in every async method on the public API (and internal too, for propagation)